tools/cmake/component.cmake

```
@@ -379,7 +379,11 @@ function(idf_component_get_property var component property)
     if(__GENERATOR_EXPRESSION)
         set(val "$<TARGET_PROPERTY:${component_target},${property}>")
     else()
-        __component_get_property(val ${component_target} ${property})
+        if("${component_target}" STREQUAL "")
+            message(FATAL_ERROR "component_target property not found for ${component} (does it need to be installed manually?)")
```

Perhaps more simply

```suggestion
            message(FATAL_ERROR "Component ${component} not found")
```

(You can give a more user-friendly message about installing the component for WolfSSL case, where you try adding WolfSSL as a dependency. "Installing manually" might not be a relevant suggestion for other components.)